spec/cspec/c/overlays/README.md

```diff
@@ -0,0 +1,22 @@
+<!--
+     Copyright 2024, Proofcraft Pty Ltd
+
+     SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# DTS Overlays
+
+This directory contains device tree overlay files that can be used to override
+default platform parameters such as available memory regions.
+
+The default files in the repository are all empty.
+
+To provide an override, place a file `overlay.dts` into the respective
+architecture directory, e.g. `ARM/overlay.dts`.
+
+The l4v build system will pick up this overlay file when it generates the
+kernel configuration data and preprocessed kernel code. It will rebuild proof
+sessions according to their dependencies.
+
+The `X64` build does not support overlays, and therefore does not provide a
+default.
```

Comment:
I like this. Question: should any other part of the docs or READMEs mention this so that people can find it?

Reply:
I couldn't find anything that seemed appropriate, but if you have a preference, happy to add it.

Reply:
The place I can think of is in `spec/cspec/c/README.md` after the paragraph "The target architecture may be specified by setting..."